### PR TITLE
Express server, API routes, PostgreSQL database complete

### PR DESCRIPTION
**Pull request by rrrliu**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7753,7 +7753,7 @@ react-dev-utils@^6.1.1:
     strip-ansi "4.0.0"
```
Comment (andrewlieu) [inline yarn.lock]: are we supposed to have yarn for server

File: server/db/groceries.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  ('first trip!'),
+  ('second trip!');
+
+INSERT INTO items
+  (trip_id, item_name)
+VALUES
+  (1, 'Pizza'),
+  (1, 'Oreos'),
+  (1, 'Cinnamon Toast Crunch'),
+  (2, 'Lube'),
```
Comment (duancaro) [inline server/db/groceries.sql:36]: owo what's this

**Comment by duancaro** [comment]
excellent job richard! superb documentation!

File: server/db/groceries.sql (hunk 2 above)
Comment (rrrliu) [reply server/db/groceries.sql:36]: lmao

**Pull request merged**